subsys/net/ip/net_private.h

```diff
@@ -28,7 +28,7 @@ union net_mgmt_events {
 #if defined(CONFIG_NET_DHCPV4)
 	struct net_if_dhcpv4 dhcpv4;
 #endif /* CONFIG_NET_DHCPV4 */
-#if defined(CONFIG_NET_L2_WIFI_MGMT)
+#if defined(CONFIG_NET_L2_WIFI_MGMT) || defined(CONFIG_NET_L2_NRF_WIFI_MGMT)
```

Comment:
This is not needed, `CONFIG_NET_L2_NRF_WIFI_MGMT` was removed.

Reply:
Ok. I'll drop the `[nrf noup] net: ip: Fix nRF Wi-Fi management events` then as this is the only line it changed.